homework/index.js

```diff
@@ -30,14 +30,120 @@
     return elem;
   }
 
+  function clearContainer(container) {
+    while (container.firstChild) {
+      container.removeChild(container.firstChild);
+    }
+  }
+
+  function renderError(err) {
+    const listContainer = document.getElementById('list-container');
+    clearContainer(listContainer);
+    const root = document.getElementById('root');
+    createAndAppend('div', root, { text: err.message, class: 'alert alert-error' });
+  }
+
+  function addRow(tbody, label, repository) {
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { text: `${label}:`, class: 'label' });
+    if (label === 'Repository:') {
+      const urlRepo = repository.html_url;
+      const td = createAndAppend('td', tr);
+      createAndAppend('a', td, {
+        text: repository.name,
+        href: urlRepo,
+        target: '_blank',
+      });
+    }
+    if (label === 'Description:') {
+      createAndAppend('td', tr, { text: repository.description });
+      console.log(repository.description);
```

Comment:
Always remove console.log statements that you have used for debugging purposes before you make a pull request.